### PR TITLE
Update version file and add tests 

### LINKED ISSUE #1355: roman-numerals: update version file and add test (1.0.0 -> 1.2.0)
Please update the version specified in the [roman-numerals](https://github.com/exercism/java/blob/master/exercises/roman-numerals/.meta/version) version file from 1.0.0 to 1.2.0.

Please also include one extra test which is included in the canonical data but not in [our tests](https://github.com/exercism/java/blob/master/exercises/roman-numerals/src/test/java/RomanNumeralsTest.java); between the tests for converting 48 and converting 59 there should be a test for converting 49.

Background info: each exercise which has canonical data should have a version file. This file states which version of the canonical data the exercise implements. The version can be found at the top of [the canonical data file for that exercise](https://github.com/exercism/problem-specifications/blob/master/exercises/roman-numerals/canonical-data.json).

### PR DESCRIPTION
**Pull request by katmpatz**
Resolves #1355 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/roman-numerals/src/test/java/RomanNumeralsTest.java [hunk 1]
```diff
@@ -69,6 +69,13 @@ public void test48ToRomanNumberXLVIII() {
         assertEquals("XLVIII", romanNumeral.getRomanNumeral());
     }
 
+    @Ignore("Remove to run test")
+	@Test
+	public void test49ToRomanNumberXLIX() {
+	    romanNumeral = new RomanNumeral(49);
+	    assertEquals("XLIX", romanNumeral.getRomanNumeral());
```
Comment (FridaTveit) [inline exercises/roman-numerals/src/test/java/RomanNumeralsTest.java]: The indentation here appears to be a bit inconsistent. I think the four statements above need to be indented with one more space 🙂

Comment (katmpatz) [reply exercises/roman-numerals/src/test/java/RomanNumeralsTest.java]: Ok!! I will fix it! 😄

**Comment by katmpatz** [comment]
I uploaded by mistake the solve of issue #1393 in this pull request. Should I have to make a new pull request for this issue?

**Comment by FridaTveit** [comment]
Yes, please only fix one issue per pull request. To make that easier it's a good idea to make a branch for each issue you start working on. Once you've pushed that branch and opened a pull request for it you can checkout master and make a new branch for the next issue you want to work on 🙂 

Does that make sense? 🙂 I know git and github can be confusing at first so let us know if you need any help 🙂

**Comment by katmpatz** [comment]
Ok I will do that! Thank you for the advice!! 😄

**Comment by FridaTveit** [comment]
No problem, that's what we're here for 🙂

**Pull request closed without merging**